integration/integration_test.py

```diff
@@ -0,0 +1,48 @@
+"""@package integration_test.py
+Entry point for our integration testing
+"""
+
+import logging
+import os
+import sys
+import unittest
+from integration_test_base import IntegrationTestBase
```

Comment:
Nit: prefer to have a separate block of imports for Python standard libs, then a block for the specific internal libs we have.

Reply:
Fixed